Bottom: 8138edb9a91f91ab79dc72d01ba48dbc8365134b
Top:    69b165e0f42557270e814f2846ac9929b97498b3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-16 23:51:41 +0000

Adjust STEP export to 2.3mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 2e2e86d..6bc2257 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -44,7 +44,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 
 HID step_hid;
